core/l10n.js

```diff
@@ -59,8 +59,10 @@ define('core/l10n',
 
     return {
         getDirection: function(context) {
+            // Try to get browser language, if undefined use default locale
+            var browserLanguage = window.navigator.language || 'en-US';
             var language = context ? context.language :
-                                     window.navigator.l10n.language;
+                                     browserLanguage;
```

Comment:
just doing 
```javascript
window.navigator.l10n.language || 'en-US'
``` 
will result in `window.navigator.l10n is undefined` because the l10n property is missing. Something like
```javascript
var language = context ? 
  context.language :
  window.navigator.l10n && window.navigator.l10n.language ?
    window.navigator.l10n.language :
    'en-US';
```
would work but i dont like it really and i think there are better ways then checking each child property?